packages/functional-tests/tests/syncV3/fxDesktopV3ForceAuth.spec.ts

```diff
@@ -120,23 +139,29 @@ test.describe('severity-1 #smoke', () => {
     });
 
     test('sync v3 with an unregistered email, registered uid', async ({
-      credentials,
+      target,
       pages: { configPage },
       syncBrowserPages: { fxDesktopV3ForceAuth, login },
+      accountManager,
     }) => {
       const config = await configPage.getConfig();
       test.skip(
         config.showReactApp.signUpRoutes === true,
         'force_auth is no longer supported for signup with react, FXA-9410'
       );
-      const email = `sync${Math.random()}@restmail.net`;
+      const account = accountManager.create();
+      const credentials = await target.createAccount(
+        account.email,
+        account.password
+      );
+      const syncAccount = accountManager.create();
```

Comment:
Needs the `SYNC_EMAIL_PREFIX` here? Might apply elsewhere too, my glazed eyes just settled upon this line.